crates/runtime/src/instance/allocator/pooling.rs

```diff
@@ -344,6 +349,15 @@ impl InstancePool {
         Ok(pool)
     }
 
+    #[cfg(target_os = "openbsd")]
+    fn new(
+        module_limits: &ModuleLimits,
+        instance_limits: &InstanceLimits,
+        tunables: &Tunables,
+    ) -> Result<Self> {
+        panic!("Pooling allocator not supported on OpenBSD due to limits in stack mmap'ing.");
```

Comment:
What kind of limits?

Reply:
MAP_STACK and issues with the way that the pooling allocator deallocs/reallocs to zero; see the commit message on the commit that added this line.

Reply:
To give a little more detail: [this check](https://github.com/openbsd/src/blob/d0b698e17cb070a8f3e5081524f33169e71390a0/sys/uvm/uvm_mmap.c#L258-L259) in the OpenBSD kernel's mmap() disallows requesting a fixed address for a MAP_STACK mmap, so one can't use this to get fresh CoW pages in an existing stack area.